requirements.txt

```diff
@@ -1,5 +1,5 @@
-pandas>=1.1.0
-numpy>=1.18
-cramjam>=2.3.0
+pandas>=1.5.0
+numpy>=1.20.3
+cramjam>=2.5.0
```

Comment:
Is there a reason for the bump to cramjam >= 2.5.0?

Tests pass with 2.3.2 for me, and I can currently not update cramjam due to https://github.com/milesgranger/pyrus-cramjam/issues/86

Reply:
Fair enough, let's try.